lib/Doctrine/DBAL/Platforms/SQLServer2008Platform.php

```diff
@@ -116,4 +116,9 @@ protected function getReservedKeywordsClass()
     {
         return Keywords\SQLServer2008Keywords::class;
     }
+
+    protected function getLikeWildcardCharacters() : iterable
+    {
+        return array_merge(parent::getLikeWildcardCharacters(), ['[', ']', '^']);
```

Comment:
`array_merge()` will not work with an arbitrary iterable. Either use `array` as the return type or use `yield from` to merge the sets.

Reply:
I'll use `array`, can't see a real need for iterable here, sorry @Taluu :P